entities/diffusion.go

```diff
@@ -24,13 +24,4 @@ type DiffusionBranch struct {
 	ShortName string `json:"shortName"`
 	CommitID  string `json:"commitIdentifier"`
 	RefType   string `json:"refType"`
-	RawFields struct {
```

Comment:
this is sometimes a struct and sometimes a list... I'll let someone who needs it deal with it